mail/clamsmtp/files/clamsmtpd.init

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh /etc/rc.common
+#
+#    (c) 2017 Lucian Cristian <lucian.cristian@gmail.com>
+#
+###########################################################################
+# CONFIGURATION
+
+# Most configuration options are found in the clamsmtpd.conf file.
+###########################################################################
+# SCRIPT
+
```

Comment:
Please drop the commented lines above and fix the style for the copyright line. Example: https://git.lede-project.org/?p=source.git;a=blob;f=package/network/services/uhttpd/files/uhttpd.init;h=6b8be1c7e4a27aa8c91f6c801af0e8c01ba9214c;hb=HEAD